build.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Copyright (c) 2020-2021, NVIDIA CORPORATION. All rights reserved.
+# Copyright 2018-2021 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Still needs to be fixed to be "2018-21". Check other places as well

Reply:
Why? Please don't add older copyright years. This file did not exist in 2018. Also, why are you removing the (c). We should just keep the existing copyright heading that we have always had but add "& AFFILICATES" and add "2021" if you are making a change this year.